src/main/webapp/search.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Search Page</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="school-name"></div>
+    <div id="login-box"></div>
+    <form id="class-form" method="POST" action="/search">
+      <p>Enter class info:</p>
+      <textarea name="class-name">Class Name</textarea>
+      <br/>
+      <textarea name="prof-name">Professor Name</textarea>
+      <br/>
+      <fieldset>      
+      <legend>How many units?</legend>      
+      <input type="radio" name="num-units" value="1">1<br>      
+      <input type="radio" name="num-units" value="2">2<br>      
+      <input type="radio" name="num-units" value="3">3<br>      
+      </fieldset> 
+      <br/><br/>
```

Comment:
Multiple br tags in a row is an indication that you should start using css to style margin/padding. Same below